app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -729,6 +799,42 @@ class AdministratorControlsActivityTest {
     fun inject(administratorControlsActivityTest: AdministratorControlsActivityTest)
   }
 
+  @Module
+  class TestModule {
+    companion object {
+      var forceEnableLanguageSelectionUi: Boolean = true
+      var forceEnableEditAccountsOptionsUi: Boolean = true
```

Comment:
Probably should be false (disabled) by default since that's the default production state.